src/encoder/rustc_serialize.rs

```diff
@@ -179,11 +179,10 @@ impl rustc_serialize::Encodable for Value {
         where E: rustc_serialize::Encoder
     {
         match *self {
-            Value::String(ref s) => e.emit_str(s),
+            Value::String(ref s) | Value::Datetime(ref s) => e.emit_str(s),
```

Comment:
Could this stay as a separate branch? It's a little clearer like that and may change one day as well